FastStyleTransfer/Demo/Helpers.swift

```diff
@@ -0,0 +1,92 @@
+import Foundation
+import TensorFlow
+import FastStyleTransfer
+
+// Make model importable
+extension TransformerNet: ImportableLayer {}
+
+func parseArgs<T>(into obj: inout T, with params: [String: WritableKeyPath<T, String?>]) {
+    for arg in CommandLine.arguments.dropFirst() {
+        if !arg.starts(with: "--") { continue }
+        let parts = arg.split(separator: "=", maxSplits: 2)
+        let name = String(parts[0][parts[0].index(parts[0].startIndex, offsetBy: 2)...])
+        if let path = params[name], parts.count == 2 {
+            obj[keyPath: path] = String(parts[1])
+        }
+    }
+}
+
+enum FileError: Error {
+    case file_not_found
```

Comment:
Use camel case for all names.
```suggestion
    case fileNotFound
```

Reply:
Done